### DISCUSSION TITLE
FUTABA

### DISCUSSION THREAD

**Discussion by m-rik-prog** [pascallanger/DIY-Multiprotocol-TX-Module#1052] [Q&A]
Has anyone ever been able to bind a Futaba receiver with the 4in1 module of a RadioMaster TX16S?

I am unable to bind Futaba's S-FHSS receiver (R2008SB) with RadioMaster's TX16S 4in1 transmitter configured with the SFHSS protocol.

I have several other Futaba receivers but they are FHSS (without “S”) and anyway without S.Bus output and T-FHSS receivers which I have therefore never tried.

From the SFHSS name used in the multimodule configuration it seems that the only Futaba receiver that can be used is the R2000SB (S-FHSS), but it does not bind.

I have searched the net but there are no posts of this possible configuration.

By any chance has any other user been able to use this one possible configuration?

Of course I have also tried different fine tuning values but always without success.

It would be helpful to know if there is any special procedure to succeed in binding between TX16S and R2008SB.

I can use the TX16S multimodule with FrSky, Spektrum, Graupner receivers but I cannot with Futaba R2008SB receivers.

Thanks!

**Comment by pascallanger** [comment]
Have you used the frequency tuning?

**Comment by m-rik-prog** [comment]
Of course! I had written it in the beginning.

Has anyone tried so far with a Futaba S-FHSS receiver (R2008SB)?

 

Da: pascallanger ***@***.***> 
Inviato: mercoledì 1 gennaio 2025 22:00
A: pascallanger/DIY-Multiprotocol-TX-Module ***@***.***>
Cc: m-rik-prog ***@***.***>; Author ***@***.***>
Oggetto: Re: [pascallanger/DIY-Multiprotocol-TX-Module] FUTABA (Discussion #1052)

 

Have you used the frequency tuning?

**Comment by DeadstickLabs** [comment]
I also haven’t been able to bind to the R2008SB. I’ve been searching for a solution and came across this thread. I guess I can give up now!
